__tests__/app.test.js

```diff
@@ -35,13 +35,72 @@ describe("GET/api", () => {
 			.get("/api")
 			.expect(200)
 			.then(({ body }) => {
-				console.log(body.endpoints);
+				// console.log(body.endpoints);
 				const parsedText = JSON.parse(body.endpoints);
 				expect(parsedText).toEqual(endpointsSource);
 			});
 	});
 });
 
+describe("GET/api/articles/:article_id", () => {
+	it("returns a single article by its Id", () => {
+		return request(app)
+			.get("/api/articles/1")
+			.expect(200)
+			.then(({ body }) => {
+				const article = body.article;
+				expect(article.article_id).toEqual(1);
+			});
+	});
+	it("returns any single article by its Id", () => {
+		return request(app)
+			.get("/api/articles/9")
+			.expect(200)
+			.then(({ body }) => {
+				const article = body.article;
+				console.log(body);
+				expect(article.article_id).toEqual(9);
+			});
+	});
+	it("returns the correct properties", () => {
+		return request(app)
+			.get("/api/articles/3")
+			.expect(200)
+			.then(({ body }) => {
+				const article = body.article;
+				console.log(article);
+				expect(typeof article.author).toBe("string");
+				expect(typeof article.title).toBe("string");
+				expect(typeof article.article_id).toBe("number");
+				expect(typeof article.body).toBe("string");
+				expect(typeof article.topic).toBe("string");
+				expect(typeof article.created_at).toBe("string");
+				expect(typeof article.votes).toBe("number");
+				expect(typeof article.article_img_url).toBe("string");
```

Comment:
One test could have done the job here. I would pick one article by id and test it, but would also test that we received the expected values rather than just testing they are the correct data type